wreleaser/README.md

```diff
@@ -0,0 +1,136 @@
+<p align="center">
+        <img src="assets/wreleaser-logo.png">
+</p>
+
+<p align="center">
+        <i><b>wreleaser</b> (WARP releaser)</i> is  a simple CLI tool for querying WARP <a href="https://github.com/broadinstitute/warp/releases">releases<a>.
+</p>
+
+## :dna: Overview
+
+WARP maintains and releases over two dozen cloud-optimized pipelines for biological data processing. Finding information for various pipeline releases via the repo's web interface is clunky and tedious.
+
+_wreleaser_ serves as a wrapper around the [GitHub Releases API](https://docs.github.com/en/rest/reference/repos#releases) specifically for WARP releases.
+
+### How does it work?
+
+On initial invocation _wreleaser_ will hit the GitHub Releases API for WARP and cache all of the releases in `$HOME/.wreleaser/cache.json`.
+
+WARP doesn't release frequently so to avoid having to hit the API regularly _wreleaser_ will return results from the cache by default.
+
+To reset your cache you can run `wreleaser reset`.
+## :gear: Installation
+
+#### Option 1
+
+Make sure you have Go installed ([download](https://jimkang.medium.com/install-go-on-mac-with-homebrew-5fa421fc55f5)). Then install _wreleaser_ with the `go get` command:
+
+```bash
+go get -u github.com/broadinstitute/warp/wreleaser@wd_GL-1606-wreleaser
+```
+
+#### Option 2
+
+We have binaries available [here](./scripts) for `linux/amd64` and `darwin/amd64`.
+
+If you have a different OS/Architecture please clone this repo and run [`go build`](https://pkg.go.dev/go/build) from this directory to compile the binary for your system.
+
+Once the binary is on your machine please add it to a location in your `$PATH`.
+## :star: Quickstart
+
+```bash
+$ wreleaser --help
+
+$ wreleaser is a lightweight CLI to list, query and download various WARP releases
+
+Currently available pipelines:
+        - AnnotationFiltration
```

Comment:
I'm not sure if we want to keep this in the list, I thought we stopped releasing it - @jessicaway or @gbggrant  do you recall?

Reply:
We stopped releasing AllOfUs (the wgs pipeline that included annotation filtration), but we still have AnnotationFiltration. As far as I know, no one is using it